Example/BeagleDemo/BeagleDemo/Constants/Constants.swift

```diff
@@ -34,6 +34,8 @@ extension String {
     static let componentsEndpoint = "/components"
     static let screenDeeplinkEndpoint = "screen-deep-link"
     static let navigateStep1Endpoint = "navigateScreenStep1"
+    static let analyticsRootIdScreen = "https://run.mocky.io/v3/6efac532-e408-422b-87a0-633eaea56d5f"
+    static let analyticsRootIdComponent = "https://run.mocky.io/v3/d77c12a3-f69f-4acf-87b1-3e6f2e0bc0fd"
```

Comment:
Could these features be added in bff? as it would be easier after automating this also using and keeping json always right

Reply:
These are just simple mocks to quick check the identifier. They are not complex at all, and used only for the inner sample app. Any other screen already added in the bff should be also of use for the same testing purposes.